src/script.js

```diff
@@ -36,13 +36,27 @@ class Item {
     removeButton.classList.add('removeButton');
     itemBox.appendChild(removeButton);
 
+    const updateButton = document.createElement('button');
+    updateButton.textContent = 'Update';
+    updateButton.classList.add('updateButton');
+    itemBox.appendChild(updateButton);
+
     editButton.addEventListener('click', () => this.edit(input));
     removeButton.addEventListener('click', () => this.remove(itemBox));
+    updateButton.addEventListener('click', () => this.update(input, itemName));
+  }
+
+  update(input, itemName) {
```

Comment:
Good work implementing the update functionality to the code

- Kindly when the user finishes updating the item ToDo, it should disable again the input. It could be pressing enter as example 🥇